src/common/CMakeLists.txt

```diff
@@ -23,6 +23,7 @@ add_library(common-lib
     factories/IdFactory.cpp
     bots/Bot.cpp
     metrics/TemporalMetric.cpp
+    models/WinCondition.cpp
```

Comment:
so we consider this a model because it's an object with some state that can be queried (using its methods)?

Reply:
models are basically just data structures for organizing data and associated functionality
vs a controller which might be organizing many different structures and be running a game loop etc
it's kind of a fuzzy definition, but this is a way of storing useful objects which might be referenced across controllers is my thinking. im mostly drawing on MVC paradigm here
https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller